src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -68,7 +68,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     // TODO: Make a better system for this. Looking for ideas!
     // sophisticated high-security admin verification system
-    if (user.getName().equals("admin")) user.adminify();
+    if (user.getName().startsWith("a")) user.adminify();
```

Comment:
what if we create attributes (user, admin)?

Reply:
I'm not sure I understand, you mean a property for users to check whether or not they are admins?

This code is for creating new admins, rather than admin checking. The comment calling it a `verification system` is inaccurate.